portal/settings.py

```diff
@@ -55,6 +55,8 @@
 
 ROOT_URLCONF = 'portal.urls'
 
+LOGIN_URL= '/users/login/'
```

Comment:
i thought i already made you guys to fix this. we need the login to be in the `/login/` not in the `users/login/`